tests/console/repo_package_install.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: check SLE repositories, packages and installation status
```

Comment:
Please phrase it SLE agnostic

Reply:
year changed to 2017

Reply:
what does agnostic mean? not really clear to me...

Reply:
Means that the test module should not mention SLE and rather be generic